R/known_outcomes.R

```diff
@@ -0,0 +1,72 @@
+#' Known outcomes from case and death timeseries data
+#'
+#' @description Calculates the number of individuals with known outcomes from a
```

Comment:
```suggestion
#' @description Estimates the expected number of individuals with known outcomes from a
```

Reply:
Fixed in https://github.com/epiverse-trace/datadelay/pull/11/commits/9131cf06d0811dc5c5145051af5b7ebd16ccad9f along with some added input checking

Reply:
To be consistent with the function-as-verb convention, we could also rename this function as estimate_known_outcomes. I think I initially called it something like this, but changed it as it was quite a long name